packages/bitcore-node/src/routes/api/block.ts

```diff
@@ -52,6 +53,34 @@ router.get('/:blockId', async function(req: Request, res: Response) {
   }
 });
 
+router.get('at-time/:time', async function(req: Request, res: Response) {
```

Comment:
Would it be possible to find an endpoint name that reflects that the resulting block's time will be  `$lte` time ?

Reply:
Changed it to `before-time` let me know if you have a better name